Replace initial notification with walkthroughs (#291)
VSpaceCode/VSpaceCode: walkthroughs/welcome/configuration.md

# Setting up

> Welcome to VSpaceCode. We will need to set up some settings and keybinginds to get started.

Reviewer: ```suggestion
Welcome to VSpaceCode. You need to set up some settings and keybindings to get started.
```
I prefer to avoid future tenses.

> 💡 Click [Configure for you](command:vspacecode.configure) or you can set up manually by reading our [documentation](command:vspacecode.openDocumentationUrl) if you want to retrain the formatting in `settings.json` and `keybindings.json`.

Reviewer: ```suggestion
💡 Click [Configure for you](command:vspacecode.configure) or you can set up manually by reading our [documentation](command:vspacecode.openDocumentationUrl) if you want to retain the formatting in `settings.json` and `keybindings.json`.
```
